torch/distributed/rpc/rref_proxy.py

```diff
@@ -31,16 +31,18 @@ def _invoke_rpc(rref, rpc_api, func_name, timeout, *args, **kwargs):
         rref.owner(),
         _invoke_func,
         args=(rref, func_name, args, kwargs),
+        device_map=device_map,
         timeout=timeout
     )
 
 # This class manages proxied RPC API calls for RRefs. It is entirely used from
 # C++ (see python_rpc_handler.cpp).
 class RRefProxy:
-    def __init__(self, rref, rpc_api, timeout=UNSET_RPC_TIMEOUT):
+    def __init__(self, rref, rpc_api, device_map, timeout=UNSET_RPC_TIMEOUT):
```

Comment:
curious, why do we need `device_map` in `RRefProxy` instead of letting the RPCs (`_invoke_rpc` above) on this proxy handling that?